File: java/src/model/ataque/RayoVeloz.java
package model.ataque;

import model.chimpos.Chinpokomon;

public class RayoVeloz extends Ataque {

    public RayoVeloz() {
        super(3, "Rayo Veloz", 1);
    }

    @Override
    public void atacar(Chinpokomon chipo, Integer valor) {
        chipo.recibirDanio(valor);
    }

}

File: python/model/ataque/RayoVeloz.py
from model.ataque.Ataque import Ataque

class RayoVeloz(Ataque):
        
        def __init__(self):
            Ataque.__init__(self, 3, "Rayo Veloz", 1)
        
        def atacar_con_valor(self, chipo, valor):
            chipo.recibir_danio(valor)